### ISSUE TITLE
Unlimited Gang Creation

### ISSUE THREAD

**Issue by HaloLVP** [LVPlayground/playground#646]
if you enter as unregistered name, you can do /gang create, after u complete all the things the server will send u a MySQL error.
![testing gang unregistered](https://user-images.githubusercontent.com/22722462/83340706-aa7bd080-a2db-11ea-8fd6-684416738be7.png)

But then after that the gang is registered to server. even if ur unregistered player still gangless.
so i dig more into it, and created like 3 gangs. 
Those gangs are in a Limbo, without a leader to disband them, but the worse thing is they are registered as official gangs in server. 
So someone can mess with and take all the possible tags and make other people unable to register their gangs with their tags.
Tz Tz2 TTZ are limbo gangs atm. if new gang arrive to LVP, they wont be able to take those tags.

Thanks for ur time

**Comment by Specifer** [comment]
Dude, Did you either mod log or RCON login? Because without these, You actually can't create gangs!
![Screenshot (192)](https://user-images.githubusercontent.com/64122067/83347172-cc5b6e80-a340-11ea-98a1-02603fc4752e.png)
![Screenshot (193)](https://user-images.githubusercontent.com/64122067/83347178-d3827c80-a340-11ea-8793-bcb2bb4f938c.png)

**Comment by HaloLVP** [comment]
didnt modlogin neither rcon, i have long time without using rcon in lvp main server.
they were characters without registration. Entered as a registered char, and then didnt login and played as a guest.

**Comment by Specifer** [comment]
Edit: yes you're right.

**Comment by RussellLVP** [comment]
Thank you for the detailed bug report! I've found the issue and this will be fixed momentarily.

Don't worry about left-over gangs, this is actually how we've structured the database. We indefinitely keep information about all past gangs, and only consider them active by virtue of having one or more active members. It avoids a bunch of synchronization, at the cost of a little bit increased data storage.
